lib/gitspindle/github.py

```diff
@@ -1524,28 +1529,28 @@ def whois(self, opts):
                 print('Location  %s' % user.location)
             if user.company:
                 print('Company   %s' % user.company)
-            print('Repos     %d public, %d private' % (user.public_repos, user.total_private_repos))
-            print('Gists     %d public, %d private' % (user.public_gists, user.total_private_gists))
+            print('Repos     %d' % user.public_repos_count)
+            print('Gists     %d' % user.public_gists_count)
```

Comment:
Did the information about private gists go away from the json data, or only the github3.py api?

Reply:
It went away in the json data. It may be a privacy thing.